doc/spinel-protocol-src/spinel-tech-thread.md

```diff
@@ -287,5 +287,65 @@ CoAP requests/responses from the NCP's RLOC address. This allows the
 host driver to implement a Thread border agent.
 
 
+### PROP 5395: PROP_THREAD_DISOVERY_REQUEST_JOINER_FLAG {#prop-thread-discovery-request-joiner-flag}
 
+* Type: Read-Write
+* Packed-Encoding:: `b`
+
+This property specifies the value used in Thread MLE Discovery Request
+TLV during discovery scan operation. Default value is `false`.
+
+### PROP 5396: PROP_THREAD_DISCOVERY_SCAN_ENABLE_FILTERING {#prop-thread-discovery-scan-enable-filtering}
+
+* Type: Read-Write
+* Packed-Encoding:: `b`
+
+This property is used to enable/disable EUI64 filtering during discovery
+scan operation. Default value is `false`.
+
+### PROP 5397: PROP_THREAD_DISCOVER_SCAN_RESULT (#prop-thread-discover-scan-result)
+
+* Type: Signal
+* Packed-Encoding:: `CcdD
+
+This property is emitted asynchronously via `CMD_PROP_VALUE_IS` to
+provide a scan result during discovery scan operation.
+
+Octets: | 1  |   1  |    2    |   *n*    |    2    |   *n*
+--------|----|------|---------|----------|---------|----------
+Fields: | CH | RSSI | MAC_LEN | MAC_DATA | NET_LEN | NET_DATA
+
+Discover result have two embedded structures which contain
+information about the MAC layer and the NET layer.
+The format below is for discover result:
+
+* `C`: Channel
```

Comment:
I could have sworn we discussed a different solution, which was adding a new scanning mode. Why the change?